File: JavaCheatSheet/src/main/java/com/github/jameswdelancey/javaCheatSheet/Part1n2.java
package com.github.jameswdelancey.javaCheatSheet;

import com.github.jameswdelancey.javaCheatSheet.part1n2.printing.TimeStuff;

public class Part1n2 {
    public static void main(String[] args) {
        System.out.println("Hello World!");
        Printing.announceTime();
        Printing.sayGoodbye();
    }
}

class Printing {
    static void announceTime() {
        TimeStuff.giveUsTheTime();
    }

    static void sayGoodbye() {
        System.out.println("Goodbye!");
    }
}

File: rustCheatSheet/src/part1n2.rs
fn main() {
    println!("Hello, world!");
    printing::announceTime();
    printing::sayGoodbye();
}

mod printing {
    pub mod timeStuff;
    pub use timeStuff::giveUsTheTime as announceTime;
    pub fn sayGoodbye() {
        println!("Goodbye!")
    }
}
